### PR TITLE
feat(docs): Updating readme to match falco.org

### PR DESCRIPTION
**Pull request by kris-nova**
Signed-off-by: Kris Nova <kris@nivenly.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

/kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

> /kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

> /area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
docs: clean up README.md to match falco.org
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,15 @@ Read the [change log](CHANGELOG.md).
 
 ---
 
-Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Falco audits a system at the most fundamental level, the kernel. Falco then enriches this data with other input streams such as container runtime metrics, and Kubernetes metrics. Falco lets you continuously monitor and detect container, application, host, and network activity—all in one place—from one source of data, with one set of rules.
+The Falco Project supports a cloud-native runtime security tool, as well as ancillary projects and integrations that surround it.
+Falco is a daemon that can run either directly on a host, or in a container.
+Falco observes system calls at runtime and builds a model of the system in memory.
+As events occur the events are parsed through a rules engine.
+If a rule is violated, an alert occurs.
+Alerts are dynamic and configurable using the Falco SDKs.
```
Comment (leodido) [inline README.md:27]: Not sure about the message of this last line Kris.

Would you please help me understand it\\what you meant here? :)

**Comment by leogr** [comment]
Since this doc update can be merged at any time without interfering with the current release, and there's no milestone set:

/milestone 0.25.0

Feel free to set this for 0.24.0 in case it get merged before

**Review by fntlnz — APPROVED** [review]

**Comment by leodido** [comment]
Closing in favor of changes Kris made with https://github.com/falcosecurity/falco/pull/1358 In August 2020.

/close

**Pull request closed without merging**